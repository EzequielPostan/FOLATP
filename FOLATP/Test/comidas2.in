loadQuery( Test/comidas.kb )
conclusion( LeGusta(John,Cacahuetes) )
runTo( Test/comidas2.out )
exit
