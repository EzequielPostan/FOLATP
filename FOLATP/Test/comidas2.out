{}
│
├─ {Mata(Cacahuetes, Bill)}
│ │
│ ├─ {Comida(Cacahuetes), Mata(Cacahuetes, Bill)}
│ │ │
│ │ ├─ {Come(Bill, Cacahuetes)}
│ │ │
│ │ └─ {Comida(x1), Mata(x1, x2), ~Come(x2, x1)}
│ │
│ └─ {~Comida(Cacahuetes)}
│   │
│   ├─ {LeGusta(John, x1), ~Comida(x1)}
│   │
│   └─ {~LeGusta(John, Cacahuetes)}
│
└─ {~Mata(Cacahuetes, Bill)}
  │
  ├─ {~Come(Bill, x1), ~Mata(x1, Bill)}
  │ │
  │ ├─ {EstaVivo(Bill)}
  │ │
  │ └─ {~Come(x1, x2), ~EstaVivo(x1), ~Mata(x2, x1)}
  │
  └─ {Come(Bill, Cacahuetes)}
